vizro-core/src/vizro/static/js/models/dashboard.js

```diff
@@ -20,7 +20,6 @@ export function _collapse_nav_panel(n_clicks, is_open) {
         false,
         {
           transform: "rotate(180deg)",
-          left: "4px",
```

Comment:
Why was this removed? 
The `left: "4px"` setting is intended to position the collapse icon so that it remains fully visible when the left side is collapsed. Without this you can only see half of the icon, or this is the intended behaviour?
We can change it to `left: "2px"` so it does not overlap with the page title.

Reply:
Without `left: "4px"` the collapse icon looks like this when collapsed. 

<img width="226" alt="Screenshot 2024-04-23 at 10 23 16" src="https://github.com/mckinsey/vizro/assets/108531476/356c9f4a-56cd-40c1-8d61-3d76633627a8">

Reply:
Yes, this is the intention that only half of the icon is visible. This is still to be discussed though. I see the advantages of both. The half-visible one doesn't change the position of the icon, so naturally "the effect" is only to collapse the side-panel without a jump in the icons position. 

I do get the reservations of the icon being only half-visible though. So this default behaviour is still to be discussed w/ J. We haven't decided on a default yet.

Reply:
We decided to keep it like that 👍